docs/_includes/admin-tasks-content.md

```diff
@@ -653,6 +653,50 @@ selected. This can be used to provide extra information about the category,
 point specific cases elsewhere (e.g. a phone number for urgent enquiries), or
 anything else you think the reporter may find useful.
 
+##### HTML content in notices
+
+Sometimes it can be helpful to use different text formatting or include images
+in the notices shown to users. You could, for example, include photos that
+illustrate the sorts of issues users should report in a particular category:
+
+<img alt="" src="/assets/img/pro-user-guide/blocked-drains-guidance.png" class="admin-screenshot" />
+
+The **Notice text** field allows you to include HTML tags to apply text
+formatting, add links, or include images in the notice shown to the user. The
+above notice was generated from this HTML:
+
+```html
+We take different action depending on how a drain is blocked.<br>
+
+<span style="font-weight: normal">A blockage on the surface of a drain could be caused by leaves or litter. These blockages can usually be cleared by carrying out street sweeping.</span><br>
```

Comment:
I wonder if we should make the extra question labels not be bold, then this sort of reset wouldn't be needed.

Reply:
Possibly, but I'd say that's beyond the scope of this PR. The example was taken from an existing cobrand so there's clearly a need there. I've ticketed this as https://github.com/mysociety/fixmystreet/issues/3142.